examples/react/start-material-ui/app/components/Header.tsx

```diff
@@ -0,0 +1,21 @@
+import { AppBar, Box, Toolbar, css, styled } from '@mui/material'
+import RouterLink from '~/components/RouterLink'
+
+const StyledRouterLink = styled(RouterLink)(
```

Comment:
Please document this usage of `StyledRouterLink` in https://github.com/TanStack/router/blob/main/docs/router/framework/react/guide/custom-link.md

Reply:
and then link to this example

Reply:
Did so! I'm not sure if I set the link properly and I don't know how to run a preview of the docs.